### ISSUE TITLE
Error on double handling in REPL

### ISSUE THREAD

**Issue by olagjo** [scriptcs/scriptcs#864]
The following series of commands seem to hit a bug in scriptcs:

```C#
var a = new double[5];
a[1] = 2.3;     //All is well so far
a[1] /= 0.5;    //prints 1.39E-309, a calculation error. Should be 4.6
a               //scriptcs hangs, and exiting returns to normal command line, now with a new and fancy font color
```

(Edit: Wording on the line with division)

**Comment by filipw** [comment]
this works with new Roslyn bits, please use this engine https://github.com/scriptcs/scriptcs-engines

**Comment by adamralph** [comment]
I guess this is probably fixed in #1064 but can we verify? Can we write test for it? Was it also a problem when executing a script or also in REPL?

**Comment by adamralph** [comment]
It's even worse on Windows 10 with .NET 4.6:

![image](https://cloud.githubusercontent.com/assets/677704/10208388/62c203ec-67d3-11e5-8c11-f375f4cf8ab7.png)

However, all is good after #1064:

![image](https://cloud.githubusercontent.com/assets/677704/10208518/164b5972-67d4-11e5-95f5-7cecdcd1178e.png)
